### ISSUE TITLE
All jobs fail in DART Version 2.1.0

### ISSUE THREAD

**Issue by rj568** [APTrust/dart#548]
When I hit Run Job nothing would happen, and it would say that the job failed due to errors but then wouldn't say what the errors were. When I went back to using 2.0 the jobs would run again

**Comment by diamondap** [comment]
Yes, unfortunately version 2.1.0 has this problem on all platforms (Windows, Mac, Linux). It's simply exiting before completing its work. I'm working on a fix.

**Comment by diamondap** [comment]
This is hard to debug. 

- All tests pass on all platforms. 
- All jobs work when run from the command line when DART is unpackaged. 
- All jobs work when run from the unpackaged DART UI. 
- All jobs work when you attach the debugger.
- No jobs work when run from the packaged version of DART created with Electron Builder. This affects all platforms.

Still working on a fix.

**Comment by diamondap** [comment]
Until this is resolved, stick with the prior version, 2.0.22.

[Mac OSX v2.0.22](https://s3.amazonaws.com/aptrust.public.download/DART/DART-2.0.22.dmg)

[Windows v2.0.22](https://s3.amazonaws.com/aptrust.public.download/DART/DART+Setup+2.0.22.exe)

[Linux v2.0.22](https://s3.amazonaws.com/aptrust.public.download/DART/DART_2.0.22_amd64.deb)
